doc/presentation.md

```diff
@@ -183,7 +183,7 @@ ie, webmaster and systeam are busy &rarr; publications have to wait
 # In an ideal world: fully automatic publication
 
 1. .green[Editor pushes changes, eg to **GitHub**]
-2. .green[CI (&ldquo;continuous integration&rdquo;) system, eg **Travis CI**, kicks in]
```

Comment:
That markdown file seems to be outdated so I suggest to remove it from the repo.